mammoth/utils/optimizers.py

```diff
@@ -32,8 +32,9 @@ def attention_bridge_optimizer(model, task_queue_manager, base_optimizer):
                 params.append(param)
             if name in suboptimizers:
                 raise Exception(f'Trying to create second optimizer for "{name}"')
-            optimizer = base_optimizer(params)
-            suboptimizers[name] = optimizer
+            if len(params) != 0:
```

Comment:
just out of curiosity: why can `params` now be empty? i am not sure how this optimizer could interact only with the embeddings

Reply:
This fixes a problem in which a sub-optimiser is created for the embedding layer but there are no trainable parameters there since the weights are frozen.